src/go/commonflags/flags.go

```diff
@@ -52,7 +52,7 @@ var (
 	//listener is marked as ready but the whole Envoy server is not marked as ready
 	//(worker did not start) somehow. To work around this problem, use IP for
 	//metadata server to fetch access token.
-	MetadataURL = flag.String("metadata_url", "http://169.254.169.254/computeMetadata", "url of metadata server")
+	MetadataURL = flag.String("metadata_url", "http://169.254.169.254", "url of metadata server")
```

Comment:
Is this flag exposed so that API Producers can override it? If so, we can't make this change. This is a breaking change if a user overrides the value. Now we'll append `/computeMetadata` to it, even if they specify it.

Please check if this was exposed. Also consider the flag default in ESPv1.

Reply:
Nop, it is not exposed in py start script.